osx/file.go

```diff
@@ -0,0 +1,144 @@
+package osx
+
+import (
+	"encoding/base64"
+	"io"
+	"net/url"
+	"os"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/pkg/errors"
+
+	"github.com/ory/x/httpx"
+)
+
+type options struct {
+	disableFileLoader   bool
+	disableHTTPLoader   bool
+	disableBase64Loader bool
```

Comment:
I would make those opt-in and not opt-out.

Reply:
> I would make those opt-in and not opt-out.

I set it explicitly to enabled because otherwise you'd always have to allow every single one. We usually load all of them, only in few instances we need some of them.

> Maybe we can also have a file-loader instance that keeps the same options around?

I think this is a good idea, if you have e.g. a load that is disabled everything per default.